.circleci/config.yml

```diff
@@ -211,7 +211,8 @@ jobs:
         command: |
           drush cc all
           drush cgen
-          drush en -y connie
+          drush en -y connie ting_covers_placeholder
+          drush dis -y ting_covers_addi
           drush vset -y ting_agency 100200
           drush vset -y opensearch_url 'https://oss-services.dbc.dk/opensearch/4.5/'
```

Comment:
Note that this url is deprecated. Latest is https://oss-services.dbc.dk/opensearch/5.0/